web/src/app/alerts/components/CreateAlertSpeedDial.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react'
```

Comment:
Let's move this file out of the components folder and have it flat under `/alerts/`, so we know that it's not legacy code when we work on the redesign. Perhaps rename `alerts/components` to `alerts/legacyComponents`?